src/Sylius/Bundle/UserBundle/Tests/Functional/app/config/config.yml

```diff
@@ -61,8 +61,6 @@ security:
     encoders:
         sylius_password_encoder_sha512:
             id: 'sylius.security.user_password_encoder.sha512'
-        sylius_password_encoder_argon2i:
```

Comment:
This is intended, after adding and committing I tried to check if argon2i is actually enabled in my sylius container. Seeing that it was not, I chose not to take any risk and remove it again.

It most likely will come back sooner or later. :)